_projects/2023/100430897/100430897.Rmd

```diff
@@ -0,0 +1,1048 @@
+---
+title: "Trust in government by race and ethnicity"
+description: |
+  Among Asian, Hispanic and Black adults, 23%, 23% and 21% respectively say they trust the federal government “most of the time” or “just about always” – higher levels of trust than among White adults (13%). During the last Democratic administration, Black and Hispanic adults similarly expressed more trust in government than White adults. Throughout most recent Republican administrations, White Americans were substantially more likely than Black Americans to express trust in the federal government to do the right thing.
+categories: "2023"
+author: Carlos San Juan Baeza
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+    self_contained: false
+    toc: true
+    
+---
+
+Distill is a publication format for scientific and technical writing, native to the web.
+
+Learn more about using Distill at <https://rstudio.github.io/distill>.
+
+ 
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(out.width="100%", fig.align="center")
+```
+
+
+
+##LIBRARIES
+
+```{r}
+
+library(ggplot2)
+library(tidyverse)
+library(readr)
+library(gridExtra)
+```
+
+##ORIGINAL PLOT##
```

Comment:
Please leave a space between the title markers and the title, and make it in sentence case, e.g. `## Some title`, without `#` after the title.